src/controller/CHIPDeviceController.h

```diff
@@ -834,13 +836,28 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
                                          /* fireAndForget = */ true);
     }
 
+    bool isNFCCommissioning(void)
```

Comment:
Please follow local style for function naming: PascalCase.  So IsNFCCommissioning().